### PR TITLE
chore(docs): add some words about Cairo 1.18

### LINKED ISSUE #3521: Dev Install Issue: Cairo must be installed from source to match CI version (1.18)
#### Preliminaries

- [x] I have followed the latest version of the
      [installation instructions](https://docs.manim.community/en/stable/installation.html).
- [x] I have checked the [installation FAQ](https://docs.manim.community/en/stable/faq/installation.html) and my problem is either not mentioned there,
      or the solution given there does not help.

## Description of error

This is more a documentation issue, but CI is using Cairo 1.18, while Ubuntu users (at least using 22.04) can only install Cairo 1.16 from `apt`.

Not using Cairo 1.18 will mean that `pytest` will skip a lot of tests locally, but that may fail in the CI. This is an issue because users will have to trust the CI to pass tests, and possibly wait a long time to check if a patch fixed changes.

## TODO

Document this potential issue (only important for dev I guess) and how to build Cairo from source (requires `ninja` and `meson`).

## System specifications

- OS: Uubuntu 22.04

### PR DESCRIPTION
**Pull request by jeertmans**
Rendered docs:

image

Closes #3521

### CODE REVIEW

File: docs/source/contributing/testing.rst [hunk 1]
```diff
@@ -5,6 +5,24 @@ If you are adding new features to manim, you should add appropriate tests for th
...
+   e.g., 1.16. If you run tests with a version prior to 1.18,
+   many tests will be skipped. Those tests are not skipped in the online CI.
+
+   If you want to run all tests locally, you need to install Cairo 1.18 or above.
+   You can do so by compiling Cairo from source:
+
+   1. download ``cairo-1.18.0.tar.xz`` from
+      `here <https://www.cairographics.org/releases/>`_.
+      and uncompress it;
+   2. open the INSTALL file and follow the instruction (you might need to install
```
Comment (chopan050) [inline docs/source/contributing/testing.rst:22]: Maybe "instructions" instead of "instruction"?

**Review by chopan050 — CHANGES_REQUESTED** [review]
Hello! Take a look at the small comment I left. Otherwise, it looks fine! Thanks for your contribution.

File: docs/source/contributing/testing.rst [hunk 2]
```diff
@@ -5,6 +5,24 @@ If you are adding new features to manim, you should add appropriate tests for th
...
+      meson and ninja);
```
Comment (chopan050) [inline docs/source/contributing/testing.rst:23]: Also, `meson` and `ninja` could be enclosed in double backticks to keep more consistency with the rest of the notes.

**Comment by jeertmans** [comment]
Nice catch @chopan050, thanks!

**Review by chopan050 — APPROVED** [review]
LGTM!

**Pull request merged**